docs/values.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Settings Values"
+date: 2020-05-04T00:00:00+00:00
+weight: 51
+geekdocRepo: https://github.com/owncloud/ocis-settings
+geekdocEditPath: edit/master/docs
+geekdocFilePath: values.md
+---
+
+A **Settings Value** is the value an authenticated user has chosen for a specific setting, defined in a
+*settings bundle*. For choosing settings values as a user the sole entry point is the ocis-web extension
+provided by this service.
+
+## Identifying settings values
+
+A *settings value* is uniquely identified by four attributes. Three of them are coming from the definition of
+the setting within it's settings bundle (see [Settings Bundles](https://owncloud.github.io/extensions/ocis_settings/bundles/)
+for an example). The fourth identifies the user.
+- extension: Key of the extension that registered the settings bundle,
+- bundleKey: Key of the settings bundle,
+- settingKey: Key of the setting as defined within the bundle,
```

Comment:
seeing it in the docs, would it make sense to either add / remove the `Key` suffix to all of them for consistency?

Reply:
I'm an advocate for removing it. I'm aware of all the bits and bobs necessary for removing it 😬

Reply:
Should we discuss this in the sprint planning? I could agree with removing `key`, but that needs PRs in phoenix, owncloud-sdk, ocis-accounts and ocis-hello as well. Definitely better to do this now, if we want to change it.

Reply:
👍 for consistency

Reply:
Raised ticket here: https://github.com/owncloud/ocis-settings/issues/32